script/download-node.js

```diff
@@ -110,7 +115,8 @@ var downloadNode = function(version, done) {
   }
 };
 
-downloadNode('v0.10.35', function(error) {
+var apmNodeVersion = process.env.APM_NODE_VERSION || '0.10.35';
```

Comment:
>We have to upgrade Node because of what I think are changes to V8 in conjunction with gyp. Basically we get module mismatches when building native modules. We could just choose to use 0.12.x but that would break Atom on master.

@thomasjo oh okay, thanks for the details.

@zcbenz so is node 0.10 unable to build native modules for Electron 0.30? Should we just start shipping 0.12 instead of 0.10 then?